client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -12,11 +12,14 @@ import VerticalNav from 'components/vertical-nav';
 import VerticalNavItem from 'components/vertical-nav/item';
 import Header from 'my-sites/upgrades/domain-management/components/header';
 import Main from 'components/main';
+import { get } from 'lodash';
 
 class Transfer extends React.Component {
 	render() {
-		const slug = this.props.selectedSite.slug,
-			domainName = this.props.selectedDomainName;
+		const slug = this.props.selectedSite.slug;
```

Comment:
In the future please keep larger janitorials in their own PRs. Changesets generally get too huge if you try to change things and tidy up all warnings.